include/libtorrent/alert_types.hpp

```diff
@@ -2355,12 +2355,27 @@ namespace libtorrent
 		int m_size;
 	};
 
+	struct TORRENT_EXPORT dht_get_peers_reply_alert: alert {
+
+		dht_get_peers_reply_alert(aux::stack_allocator& alloc
+			, sha1_hash const& ih
+			, std::vector<tcp::endpoint> const& v);
+
+		const static int static_category = alert::dht_notification;
+		TORRENT_DEFINE_ALERT(dht_get_peers_reply_alert, 87)
+
+		virtual std::string message() const;
+
+		sha1_hash info_hash;
+		std::vector<tcp::endpoint> peers;
```

Comment:
do you think it would be possible to allocate this using aux::stack_allocator instead? (i.e. m_alloc). It would be a lot more efficient